tests/formats/test_bmsld.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+from tests.test_lib import parse_build_compare_editor
+
+from mercury_engine_data_structures.formats.bmsld import Bmsld
+
+all_bmsld = [
+    "maps/levels/c10_samus/s000_surface/s000_surface.bmsld",
+    "maps/levels/c10_samus/s010_area1/s010_area1.bmsld",
+    "maps/levels/c10_samus/s020_area2/s020_area2.bmsld",
+    "maps/levels/c10_samus/s025_area2b/s025_area2b.bmsld",
+    "maps/levels/c10_samus/s028_area2c/s028_area2c.bmsld",
+    "maps/levels/c10_samus/s030_area3/s030_area3.bmsld",
+    "maps/levels/c10_samus/s033_area3b/s033_area3b.bmsld",
+    "maps/levels/c10_samus/s036_area3c/s036_area3c.bmsld",
+    "maps/levels/c10_samus/s040_area4/s040_area4.bmsld",
+    "maps/levels/c10_samus/s050_area5/s050_area5.bmsld",
+    "maps/levels/c10_samus/s060_area6/s060_area6.bmsld",
+    "maps/levels/c10_samus/s065_area6b/s065_area6b.bmsld",
+    "maps/levels/c10_samus/s067_area6c/s067_area6c.bmsld",
+    "maps/levels/c10_samus/s070_area7/s070_area7.bmsld",
+    "maps/levels/c10_samus/s090_area9/s090_area9.bmsld",
+    "maps/levels/c10_samus/s090_area9/s090_area9.bmsld",
+    "maps/levels/c10_samus/s100_area10/s100_area10.bmsld",
+    "maps/levels/c10_samus/s110_surfaceb/s110_surfaceb.bmsld",
+
+]
+
+
+@pytest.fixture()
+def surface_bmsld(samus_returns_tree) -> Bmsld:
+    return samus_returns_tree.get_parsed_asset(all_bmsld[0], type_hint=Bmsld)
+
+@pytest.mark.parametrize("bmsld_path", all_bmsld)
+def test_bmsld(samus_returns_tree, bmsld_path):
+    parse_build_compare_editor(Bmsld, samus_returns_tree, bmsld_path)
+
+def test_all_actor_groups(surface_bmsld: Bmsld):
+    all_groups = surface_bmsld.all_actor_groups()
+    assert len(list(all_groups)) == 32
+
+@pytest.mark.parametrize("cc_name,actor_name,should_be_present",
```

Comment:
I much prefer the alternate syntax of `("cc_name", "actor_name", "should_be_present")` here rather than the comma separated string